src/services/completions.ts

```diff
@@ -153,6 +153,11 @@ namespace ts.Completions {
             return stringCompletions;
         }
 
+        const objectTypeLiteralCompletions = getObjectTypeLiteralInTypeArgumentCompletions(sourceFile, position, contextToken, typeChecker, compilerOptions, log, preferences);
+        if (objectTypeLiteralCompletions) {
+            return objectTypeLiteralCompletions;
+        }

```

Comment:
Instead of putting this here, did you look at adding to `getCompletionData` -> `tryGetGlobalSymbols`? It feels similar to `tryGetObjectLikeCompletionSymbols()` and `tryGetClassLikeCompletionSymbols()` so I expected it to be another case in that big chain of logical ORs.

Reply:
I did look into it initially, but wasn't sure whether it would be the best place to put it. However, I tried moving the code there, and it still works as expected, so I think it should be fine :-)